src/main/resources/jdbc/jdbc.properties

```diff
@@ -1,8 +1,8 @@
 driver=org.postgresql.Driver
-url=jdbc:postgresql://localhost/EpamMarket
-name=EpamMarket
-userName=admin
-password=postgres
+url=jdbc:postgresql://localhost/epammarket
+name=epammarket
+userName=postgres
+password=admin
 host=localhost
 port=5432
 pool.max=50
```

Comment:
Looks clear, no comments. But I would ask you to move some trash from project within this task. I'm talking about folders "out", "build" and "settings". They are already in .ignore file, but I keep seeing them on github.